openedx/core/djangoapps/agreements/serializers.py

```diff
@@ -31,3 +32,10 @@ class LTIPIISignatureSerializer(serializers.ModelSerializer):
     class Meta:
         model = LTIPIISignature
         fields = ('username', 'course_id', 'lti_tools', 'created_at')
+
+
+class UserAgreementsSerializer(serializers.Serializer):
+    """
+    Serializer for UserAgreementRecord model
+    """
+    accepted_at = serializers.DateTimeField()
```

Comment:
Why not include user and agreement_type too ?
I realize the user and agreement type are infered from the request itself, but I feel that when we are going teh GET request, we are fetching a record of the user agreement and not just the timestamp(s) of when the user agreed. Including the username and agreement type would make the fetched record complete.

What do you think ?

Reply:
Sure, that makes sense.